src/PostgREST/App.hs

```diff
@@ -105,6 +105,19 @@ app conf reqBody req =
             , (hLocation, "/postgrest/users?id=eq." <> cs (userId u))
             ] ""
 
+    (["postgrest", "users"], "PUT") -> do
```

Comment:
The PUT verb should probably apply directly to an individual user resource, i.e. `/postgrest/users?id=eq.n` rather than `/postgrest/users`.